rfcs/0085-nixos-release-stablization.md

```diff
@@ -0,0 +1,128 @@
+branch-off.
+
+This happened for the 20.09 release in which ~640 staging
+commits were pushed through in the days leading up to the branch off
+([#95492](https://github.com/NixOS/nixpkgs/pull/95492),
+[#96280](https://github.com/NixOS/nixpkgs/pull/96280),
+[#96437](https://github.com/NixOS/nixpkgs/pull/96437),
+[#97146](https://github.com/NixOS/nixpkgs/pull/97146)). The last PR included a
+release blocker fix (introduced in [one of the mentioned
+PRs](https://github.com/NixOS/nixpkgs/pull/96437)) which delayed the
+branch-off from 4 Sept 2020 to 7 Sept 2020, and ZHF had to be delayed
+another day to get a semi-accurate hydra build status. The systemd
+bump was also largely responsible delaying the release until after the
+scheduled release date where basic login features in plasma [weren't
+restored until Oct 5th, a week after the planned release
+date](https://github.com/NixOS/nixpkgs/pull/99629) and a backlog of
+other plasma related issues [weren't resolved until Oct 19th, almost a
+month after the planned release
+date](https://github.com/NixOS/nixpkgs/pull/101078).  This is not to
+discredit any individual, but demonstrate that certain packages have
+many dimensions of compatibility which need a longer time to stabilize
+on unstable before being included in a release.
+
+
+# Definitions
+[definitions]: #definition
+- Breaking change
+  - A change that is likely to break downstream.
+  - For projects following Semantic Versioning, disallow major version bumps.
+- Restricted
+  - Breaking changes are disallowed.
+  - Breaking changes for security or critical bugs are allowed. (Patching is preferred over updates.)
+  - New packages and NixOS modules are welcome.
+- Unrestricted
+  - Breaking changes are allowed.
+  - New packages and NixOS modules are welcome.
+- ZHF: Zero Hydra Failures
+  - Period in which packages are stabilized in preparation of a release.
+
+# Changes
+
+## Release Critical Packages
+
+A list of Release Critical Packages will be maintained in the nixpkgs
+manual.  Release Critical Packages are important packages which have
+many dimensions of build or runtime behavior. Generally these will be
+packages which are a part of `stdenv`, or bootstrapping a system
+(e.g. systemd). The Release Managers are empowered to decide which
+packages are Release Critical Packages without additional RFCs.
+
+Initially these will be `stdenv.cc`,`binutils`, and `systemd`.
+
+## Restriction Timeline
+
+This timeline represents when certain branches and packages will be
+restricted. It is not a complete timeline of all release activities.
+The actual timeline will still be determined by the Release Managers
+and may be adjusted as needed. The current position in the timeline
+will be communicated on Discourse.
+
+| Weeks from Release | Branches Affected | Events |
+| --- | --- | --- |
+| -8 Weeks | | Gnome and Plasma(YY.11) packaging begins |
+| -6 Weeks | `staging-next`, `staging` | Restrict breaking changes to Release Critical Packages |
```

Comment:
Since we defined Restricted above, I think we can just say:

```suggestion
| -6 Weeks | `staging-next`, `staging` | Release Critical Packages are Restricted |
```